guide/onboarding/manual-backup.md

```diff
@@ -0,0 +1,264 @@
+
+“Your recovery phrase is a group of 12 random words”
+
+{% include dl/item-middle.html color="orange" %}
+
+Explains to users what a recovery phrase **is**.
+
+{% include dl/item-close.html %}
+
+{% include dl/item-open.html color="yellow" %}
+
+“Your recovery phrase is the only way to acess your wallet if your phone is lost or stolen.”
+
+{% include dl/item-middle.html color="yellow" %}
+
+Explains to users what the **purpose** of a recovery phrase is and why it’s important.
+
+{% include dl/item-close.html %}
+
+{% include dl/item-open.html color="green" %}
+
+“If you lose your recovery phrase, you will no longer be able to access your wallet. Never share your recovery phrase with anyone. Anyone who has it can access your funds.”
+
+{% include dl/item-middle.html color="green" %}
+
+Explains to users what the **consequences** of their behavior is, and how it can affect the safety of their funds.
+
+{% include dl/item-close.html %}
+
+{% include dl/item-open.html %}
+
+“We recommend writing these words down in order on a piece of paper and storing it somewhere safe that you will remember.”
+
+{% include dl/item-middle.html %}
+
+Guides and gives users actionable items on how to safely **handle** their recovery phrase.
+
+{% include dl/item-close.html %}
+
+{% include dl/close.html %}
+
+Drilling in the larger consequences of what it means to interact with a self-custodial product is important during these first interactions with a wallet. Education within these beginning stages will empower users to make smart decisions, furthermore informing how they understand and handle the safety of their funds.
+
+{% include /tip/open.html label="Tip: Give users access to printable templates" icon="info" color="blue" %}
+
+Providing users with printable, designated materials to write down their recovery phrase can instill a sense of trust, organization (especially if they have multiple wallets), and guidance. It may also encourage them to treat this designated paper with importance, rather than quickly scribbling it down on a random scrap. Check out an example template from the Wallet UI Kit [here](https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-Wallet-UI-Kit?node-id=1523%3A34106).
+
+{% include /tip/close.html %}
+
+### Ways to show it
+
+The goal is to encourage users to write it down their recovery phrase on a physical piece of paper. You can discourage users from taking a screenshot by showing a warning modal, or disabling the functionality altogether.
+
+{% include /tip/open.html label="Tip: Be Clear about Numbering" icon="info" color="blue" %}
+
+Regardless of the option you decide to run with, it’s important that you explicitly instruct users to number each word (i.e. 1. sand 2. purple 3. flower). Manually backing up recovery phrases typically asks users to recount, for example, the fifth word. It’s a bit of a pain for users to count which word corresponds to a particular number if they are not numbered initially.
```

Comment:
```
Regardless of the option you decide to run with, it’s important that you explicitly instruct users to number each word (i.e. 1. sand 2. purple 3. flower). Typically, a manual recovery phrase backup scheme will ask users to recount, for example, the fifth word. It’s a bit of a pain for users to count which word corresponds to a particular number if they are not numbered initially.
```

Reply:
The sentence might have been grammatically correct, but somethings sounds awkward to me about it.